sdks/wasm/Makefile

```diff
@@ -229,6 +229,18 @@ bin/test-suite/mono.js: packager.exe $(WASM_FRAMEWORK)/.stamp-framework binding_
 
 build-test-suite: bin/test-suite/mono.js
 
+xunit-main.exe: xunit-main.cs
+	csc /out:$@ /r:xunit.abstractions.dll /r:xunit.core.dll /r:$(WASM_BCL_DIR)/Facades/System.Runtime.dll /r:$(WASM_BCL_DIR)/Facades/System.Reflection.dll /r:$(WASM_BCL_DIR)/Facades/System.Collections.dll /r:xunit.execution.dotnet.dll /r:xunit.runner.utility.net452.dll xunit-main.cs
```

Comment:
maybe add
XUNIT_DIR=$(TOP)/external/xunit-binaries/
for the xunit dll references?